packages/assets-controllers/src/NftDetectionController.ts

```diff
@@ -401,24 +609,23 @@ export class NftDetectionController extends StaticIntervalPollingControllerV1<
     }
 
     const apiNfts = await this.getOwnerNfts(userAddress);
-    const addNftPromises = apiNfts.map(async (nft: ApiNft) => {
+    const addNftPromises = apiNfts.map(async (nft: TokensResponse) => {
```

Comment:
TypeScript should be able to infer this type. What happens if you leave it off? If this produces a type error, does `getOwnerNfts` return the correct type?

Reply:
If i remove the type i do not get any errors. I have added because we were specifying the type ApiNft in the older version. I can remove it so its: 
`const addNftPromises = apiNfts.map(async (nft) => {`